bin/theme

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -e
+
+read -p "Please enter your new WordPress theme name: " ORIGINAL
+
+HYPHENATED=`echo "${ORIGINAL}" | tr " " -`
+LOWERCASE=`echo "${HYPHENATED}" | tr '[A-Z]' '[a-z]'`
+UPPERCASE=`echo "${HYPHENATED}" | tr '[a-z]' '[A-Z]'`
+TITLECASE=`echo "${HYPHENATED}" | tr '-' '\n' | awk '{printf "%s%s", toupper(substr($0,0,1)), substr($0,2)}'`
+
+echo ""
+
+replace_lowercase() {
+  eval `sed -i '' "s/skela/${LOWERCASE}/" .env`
+  eval `sed -i '' "s/skela/${LOWERCASE}/" .env.sample`
+  eval `sed -i '' "s/skela/${LOWERCASE}/" composer.json`
+  eval `sed -i '' "s/skela/${LOWERCASE}/" composer.lock`
+  eval `sed -i '' "s/skela/${LOWERCASE}/" package-lock.json`
+  eval `sed -i '' "s/skela/${LOWERCASE}/" package.json`
+  eval `sed -i '' "s/skela/${LOWERCASE}/" webpack.config.js`
+  eval `sed -i '' "s/skela/${LOWERCASE}/" bin/composer`
+  eval `sed -i '' "s/skela/${LOWERCASE}/" src/Blocks/SampleBlock/sample-block.js`
+  eval `sed -i '' "s/skela/${LOWERCASE}/" src/Managers/ThemeManager.php`
+  eval `sed -i '' "s/skela/${LOWERCASE}/" templates/pages/front-page.twig`
+}
+
+replace_uppercase() {
+  eval `sed -i '' "s/SKELA/${UPPERCASE}/" functions.php`
+  eval `sed -i '' "s/SKELA/${UPPERCASE}/" src/**/*.php`
+}
+
+replace_titlecase() {
+  eval `sed -i '' "s/Skela/${TITLECASE}/" .env`
+  eval `sed -i '' "s/Skela/${TITLECASE}/" .env.sample`
+  eval `sed -i '' "s/Skela/${TITLECASE}/" composer.json`
+  eval `sed -i '' "s/Skela/${TITLECASE}/" front-page.php`
+  eval `sed -i '' "s/Skela/${TITLECASE}/" functions.php`
+  eval `sed -i '' "s/Skela/${TITLECASE}/" index.php`
+  eval `sed -i '' "s/Skela/${TITLECASE}/" style.css`
+  eval `sed -i '' "s/SKELA/${TITLECASE}/" src/**/*.php`
+}
+
+echo "Replacing 'skela' with '$LOWERCASE'"
+replace_lowercase
+
+echo "Replacing 'SKELA' with '$UPPERCASE'"
+replace_uppercase
+
+echo "Replacing 'Skela' with '$TITLECASE'"
+replace_titlecase
```

Comment:
There's a missing case here -- multi-word theme names!

For example, if I set a theme name of `The 19th News` I get the following in `.env`:

```
WORDPRESS_TITLE="The19thNews Site"
UPS_DOCK_NAME=The19thNews
```

When I actually want:

```
WORDPRESS_TITLE="The 19th News"
UPS_DOCK_NAME=The 19th News
```

I think the sed search strings may be a bit too broad.